### PR TITLE
Ubuntu 22.04 STIG V2R1 changes

### PR DESCRIPTION
**Pull request by ericeberry**
#### Description:

- Changes to reflect the updates in Ubuntu 22.04 STIG V2R1

#### Rationale:

- Part of Ubuntu 22.04 DISA STIG v1r12 profile upgrade

#### Review Hints:

- Build the product:
./build_product ubuntu2004

- To test these changes, please ensure that the remediation headers are updated to V2R1.

- The Manual STIG is the current latest version, and was obtained from DISA's website. For reference, please review: https://public.cyber.mil/stigs/downloads/

### CODE REVIEW

File: controls/srg_gpos/SRG-OS-000366-GPOS-00153.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ controls:
             - ensure_gpgcheck_globally_activated
             - ensure_gpgcheck_local_packages
             - ensure_gpgcheck_never_disabled
+            - apt_conf_disallow_unauthenticated
```
Comment (dodys) [inline controls/srg_gpos/SRG-OS-000366-GPOS-00153.yml:15]: try to add this line as
```
{{% if 'ubuntu' in product %}}
- apt_conf_disallow_unauthenticated
{{% endif %}}
```
that should make build pass.

File: controls/srg_gpos/SRG-OS-000356-GPOS-00144.yml [hunk 2]
```diff
@@ -8,4 +8,5 @@ controls:
         rules:
             - var_time_service_set_maxpoll=18_hours
             - chronyd_or_ntpd_set_maxpoll
+            - chronyd_sync_clock
```
Comment (dodys) [inline controls/srg_gpos/SRG-OS-000356-GPOS-00144.yml:11]: try to add this line as
```
{{% if 'ubuntu' in product %}}
- chronyd_sync_clock
{{% endif %}}
```
that should make build pass.

**Review by dodys — CHANGES_REQUESTED** [review]

File: products/ubuntu2204/profiles/stig.profile [hunk 3]
```diff
@@ -26,7 +26,7 @@ selections:
     # UBTU-22-612040 The Ubuntu operating system must map the authenticated identity to the user or group account for PKI-based authentication.
     - verify_use_mappers
 
-    # UBTU-22-411025 The Ubuntu operating system must enforce 24 hours/1 day as the minimum password lifetime. Passwords for new users must have a 24 hours/1 day minimum password lifetime restriction.
+    # UBTU-22-411025 The Ubuntu operating system must enforce 24 hours/one day as the minimum password lifetime. Passwords for new users must have a 24 hours/one day minimum password lifetime restriction.
```
Comment (dodys) [inline products/ubuntu2204/profiles/stig.profile:29]: on line 3, please update the version too as  well as adding the manual to shared/references.

**Review by dodys — CHANGES_REQUESTED** [review]
a few things to include

File: products/ubuntu2204/profiles/stig.profile (hunk 3 above)
Comment (ericeberry) [reply products/ubuntu2204/profiles/stig.profile:29]: Done

**Review by dodys — APPROVED** [review]
lgtm, thanks!

**Pull request merged**